nats-concepts/what-is-nats/walkthrough_setup.md

```diff
@@ -14,6 +14,72 @@ For MacOS:
 brew tap nats-io/nats-tools
 brew install nats-io/nats-tools/nats
 ```
+# Setting Up NATS Server in Your PATH
+
+This guide will help you add the NATS server to your PATH for both current and future terminal sessions.
+
+## Step 1: Add NATS Server to PATH
+
+First, you need to append the NATS server directory to your PATH variable. This can be done by adding the following line to your `.zshrc` file. You can use a text editor like `vi` for this purpose:
+
+```sh
+vi ~/.zshrc
+```
+In the editor, add the following line at the end of the file:
+```
+export PATH=\$PATH:~/usr/local/Cellar/nats/0.1.4/bin
```

Comment:
this is not neccesary, the reason you have to do this is because your homebrew setup is incorrect.  Correctly install homebrew and this will just work 

These paths are also very unusual and probably due to something that went wrong with your homebrew